### DISCUSSION TITLE
Validation loss is decreasing but WER is increasing in Whisper model training?

### DISCUSSION THREAD

**Discussion by monk1337** [openai/whisper#1711] [Q&A]
Hi, I've been using the Huggingface library to fine-tune the Whisper model. While the WER was initially decreasing, I've noticed it began to rise even though the validation loss continues to drop. Could the issue be related to my testing on a very small dataset?

As shown in the image, after 80th step the wer suddenly started increasing from 13 -> 28

<img width="419" alt="Screenshot 2023-10-14 at 4 28 50 PM" src="https://github.com/openai/whisper/assets/17107749/12272fb8-823d-4762-894b-da3f8a427f4d">
